libs/transactions/src/lib/extensions/transaction-addition-details.ts

```diff
@@ -0,0 +1,14 @@
+import {InjectionToken, Type} from "@angular/core";
+import { TransactionItem } from "@backbase/data-ang/transactions";
+import {ExtensionComponent} from "../extension-slot.directive";
+
+export type TransactionAdditionalDetailsContext = Pick<TransactionItem, 'additions' | 'counterPartyAccountNumber' | 'merchant'>;
+export type TransactionAdditionalDetailsComponent = ExtensionComponent<TransactionAdditionalDetailsContext>;
+
+export interface TransactionsJourneyExtensionsConfig {
```

Comment:
since this config is shared, containing all extension slots, I assume it doesn't belong in this (extension specific) file